lib/config/config-initializer.js

```diff
@@ -239,43 +242,61 @@ function configureRules(answers, config) {
  * @returns {Object} config object
  */
 function processAnswers(answers) {
-    let config = { rules: {}, env: {}, parserOptions: {} };
+    let config = {
+        rules: {},
+        env: {},
+        parserOptions: {},
+        extends: []
+    };
 
-    config.parserOptions.ecmaVersion = answers.ecmaVersion;
-    if (answers.ecmaVersion >= 2015) {
-        if (answers.modules) {
-            config.parserOptions.sourceType = "module";
-        }
-        config.env.es6 = true;
-    }
+    // set the latest ECMAScript version
+    config.parserOptions.ecmaVersion = DEFAULT_ECMA_VERSION;
+    config.env.es6 = true;
```

Comment:
ES2017 has added two globals: `Atomics` and `SharedArrayBuffer`. Because now `DEFAULT_ECMA_VERSION` is `2018` and `env.es6` doesn't include those, should we add `globals.Atomics` and `globals.SharedArrayBuffer` here?

Reply:
Oh yeah, that's a good idea. Thanks.